### DISCUSSION TITLE
How to add a private world link to VRCX favorites?

### DISCUSSION THREAD

**Discussion by everruler12** [vrcx-team/VRCX#525] [Q&A] [answered]
Let's say I have a direct link to a private world, like https://vrchat.com/home/world/wrld_b268b844-c56f-42c5-9199-94d809622f84

And then I want to add it to Local Favorites in a "Worlds to visit" group. How can I do this?

I've tried doing a search in VRCX for the link and the world ID, but the world doesn't appear.

My current method is to launch/invite myself from the VRC website. Then the world would show up under my VRCX Game Log. But this requires me to open the game and actually visit the world first (defeating the purpose of adding it to "Worlds to visit" for me to check out the next time I open the game)

**Comment by Myrkie** [comment, marked as answer]
you should beable to just add it to your local favorites without issues, adding it to your normal vrchat favorites for private worlds will result in an error just, by copying the ID and then using "direct access" then adding to a local group should function fine

[firefox_DGlnsHrkFE.webm](https://user-images.githubusercontent.com/20288698/233517170-dd3ef287-9a1a-478e-8e21-9dc514a2fed2.webm)



![image](https://user-images.githubusercontent.com/20288698/233517077-12ca7403-df59-40f6-aec1-29c98df5f710.png)

**Reply by everruler12** [reply]
Perfect! I never used that button before. Thanks

**Reply by Natsumi-sama** [reply]
You can also paste full URLs in there.
